packages/teams-js/src/public/visualMedia.ts

```diff
@@ -0,0 +1,274 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { DevicePermission } from './interfaces';
+import { runtime } from './runtime';
+
+/**
+ * Interact with image and video. It lets the app developer ask the user to get images or video from their camera / camera roll / file system.
+ * @beta
+ */
+export namespace visualMedia {
+  /**
+   * @hidden
+   * Hide from docs
+   * --------
+   * All properties common to Image and Video Props
+   * @beta
+   */
+  interface VisualMediaProps {
+    /**
+     * @hidden
+     * Lets the developer specify the media source
+     */
+    source: Source;
+
+    /**
+     * @hidden
+     * Optional; Specify in which mode the camera will be opened.
+     * Default value is Photo
+     */
+    startMode?: CameraStartMode;
+
+    /**
+     * @hidden
+     * Optional; indicate if user is allowed to move between front and back camera
+     * Default value is true
+     */
+    cameraSwitcher?: boolean;
```

Comment:
I see that the comments say the default value is true, but that doesn't indicate which camera will be used in that case. Is the default camera the front or the back?

Consider making this an enum instead to make the usage clearer. 
```
enum CameraRestriction {
 frontOrRear,
 frontOnly,
 rearOnly,
}
```

(or if not all three options are available, then only the two that are available)